guides/common/modules/con_managing-ostree-content.adoc

```diff
@@ -0,0 +1,17 @@
+[id="con_managing-ostree-content_{context}"]
+= Managing OSTree Content
+
+*OSTree* is a tool to manage bootable, immutable, versioned file system trees.
+
+TODO why
```

Comment:
I feel that you don't necessarily need to answer this question. 
We don't need to convince anyone to manage OSTree content in Katello. 
We just need to tell them how.